Assets/MRTK/Examples/Demos/HandTracking/Scripts/LeapMotionOrientationDisplay.cs

```diff
@@ -4,7 +4,7 @@
 using UnityEngine;
 using Microsoft.MixedReality.Toolkit.Input;
 using TMPro;
-#if LEAPMOTIONCORE_PRESENT
+#if (LEAPMOTIONCORE_PRESENT && UNITY_STANDALONE) || (LEAPMOTIONCORE_PRESENT && UNITY_WSA && UNITY_EDITOR)
```

Comment:
In the latest Leap Motion release (https://developer.leapmotion.com/releases/leap-motion-orion-410), the notes mention

> Allow access to Leap Motion tracking data from UWP and AppContainer applications

Do we want to instead make our data provider compatible with runtime UWP?

Reply:
I can investigate, follow up task noted here: #10038